privly.safariextension/Info.plist

```diff
@@ -3,9 +3,9 @@
 <plist version="1.0">
```

Comment:
The plist version stays `1.0`. The version of the extension is defined by `CFBundleShortVersionString` and `CFBundleVersion` which has been changed to `0.1.0`